.github/workflows/ci.yml

```diff
@@ -39,6 +39,7 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install mypy==0.800
+          pip install pytest==6.1.2
```

Comment:
Couldn't we just upgrade the dependency in our requirements.txt ?

Reply:
This job does not install any other packages from requirements